packages/effect/src/Fiber.ts

```diff
@@ -275,7 +275,8 @@ export {
  * @since 2.0.0
  * @category destructors
  */
-export const awaitAll: (fibers: Iterable<Fiber<any, any>>) => Effect.Effect<void> = fiberRuntime.fiberAwaitAll
+export const awaitAll: <A, E>(fibers: Iterable<Fiber<A, E>>) => Effect.Effect<Array<Exit.Exit<A, E>>> =
```

Comment:
Hmm, this is a major departure from the original implementation.
I wonder why this was implemented as `await(fiberAll(fibres))`, instead of `forEach(fibers, Fiber.await)`

Reply:
any failing tests?

Reply:
Just the one I updated which was expecting the void result.

Reply:
then I think it is fine to change the behaviour, the old one is odd

Reply:
Technically this is a breaking change. Did we still want to keep this as a minor changeset?

Reply:
Technically it's also a bug fix. hmm

Reply:
@mikearnaldi I'm ok with this change as minor. What do you think?